config/software/procps-ng.rb

```diff
@@ -18,9 +18,10 @@
   ship_source "http://dd-accgent-omnibus.s3.amazonaws.com/#{name}-#{version}.tar.xz"
   ship_license "https://gitlab.com/procps-ng/procps/raw/master/COPYING"
   command(["./configure",
-     "--prefix=#{install_dir}/embedded",
-     ""].join(" "),
-  :env => env)
+           "--prefix=#{install_dir}/embedded",
+           "--without-ncurses",
```

Comment:
So in [line 4](https://github.com/DataDog/omnibus-software/pull/72/files#diff-ff9ac6b4c018b929f0bf85b3872faf2aR4) we have `ncurses` as a dependency but we then don't use it... I was also wondering if we should only build without ncurses if it's a `suse` build? (This last part is just a thought).

If we decide to drop `ncurses` dependency, and there are no other deps, we can maybe remove that from `config/software/`.